tests/Utils/Policies/PostPolicy.php

```diff
@@ -11,4 +11,9 @@ public function view(User $user, Post $post): bool
     {
         return $post->user_id === $user->getKey();
     }
+
+    public function delete(User $user, Post $post): bool
+    {
+        return (int)$post->user_id === (int)$user->getKey();
```

Comment:
The typecasting isn't working for SQLite 🤔

Reply:
It's fine